dpe/afl/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+# Licensed under the Apache-2.0 license
+
+[package]
+name = "dpe-afl"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[dependencies]
+afl = "*"
+log = "*"
+simplelog = "*"
```

Comment:
Prefer specific versions so everyone is running tests with the same environment.

Reply:
I specified versions of libraries used internally, such as `openssl`, I wasn't sure there was a point here. Maybe?

Reply:
In general we prefer to specify specific versions so that we ensure the every developer + the CI are running the same code. It helps with reproducing any issues. So I think we should specify the latest version for all of these.

Reply:
Done, will push shortly.